src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
-spring.application.name=learning-management-system
```

Comment:
TODO:
This application.properties should be replaced with the one from the db-connection branch.
I am not sure if we can delete it here from the review, will check and add another comment.